FlowCrypt/Controllers/Threads/ThreadDetailsViewController+TapActions.swift

```diff
@@ -153,6 +159,28 @@ extension ThreadDetailsViewController {
         let attachments = quoteType == .forward
             ? input.processedMessage?.attachments ?? []
             : []
+        showSpinner()
+        let decryptedAttachments = try await attachments.asyncMap { (attachment: MessageAttachment) -> MessageAttachment in
```

Comment:
Probably `MessageAttachment` declaration not needed here, as swift compiler will get it automatically